package.json

```diff
@@ -48,15 +48,16 @@
         "jsesc": "2.5.2",
         "morgan": "^1.9.1",
         "newrelic": "^5.9.0",
-        "react": "^16.8.4",
-        "react-dom": "^16.8.4"
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1"
```

Comment:
I kept running into warnings when I ran my Jest tests and the Internet said that upgrading to the latest would fix it, so this and the Jest version bump a few lines below are a result of that.

Reply:
This means we now have to test more carefully on everything since, sadly, this is a pretty severe change. 
We just need to make a mental note of testing everything more careful before merging this PR.

Reply:
I know I don't have as much historical context, but I don't see this as a severe change, since they are minor updates, which are inherently assumed to be additive and non-breaking. From a quick glance at the React change log between 16.8.4 and latest, I don't see any glaring issues, but happy to test the rest of the React site more thoroughly.